Dockerfile.ci

```diff
@@ -7,4 +7,9 @@ RUN pyenv install -s 3.6.13
 RUN pyenv install -s 3.7.1
 RUN pyenv install -s 3.7.10
 
-RUN git config --global user.name dependabot-ci && git config --global user.email no-reply@github.com
+RUN git config --global user.name dependabot-ci && \
+  git config --global user.email no-reply@github.com
+
+ARG CODE_DIR=/home/dependabot/dependabot-core
+WORKDIR ${CODE_DIR}
+COPY --chown=dependabot:dependabot . ${CODE_DIR}
```

Comment:
Thoughts on splitting this up to more granular COPYs? This step takes over 2 mins on my machine and I guess would always run in full if any file changes?

Reply:
Tried this but ran into a `max depth exceeded` error after 104 layers, if we copy all folders inside each pm we get 117 😬

Reply:
SGTM!

Reply:
Even after reducing the layers to 76 it looked like actions gave up and killed the process so went back to copying it all in one go https://github.com/dependabot/dependabot-core/runs/2297918989